core/src/main/scala/filodb.core/memstore/PartKeyLuceneIndex.scala

```diff
@@ -1220,7 +1224,7 @@ class PartIdStartTimeCollector extends SimpleCollector {
   }
 }
 
-class PartKeyRecordCollector extends SimpleCollector {
+class PartKeyRecordCollector(limit: Int) extends SimpleCollector {
   val records = new ArrayBuffer[PartKeyLuceneIndexRecord]
```

Comment:
Same as before for pre-initialing with size to avoid reallocating space.

Reply:
for now leaving it as is